### PR TITLE
Fix multiple file labels in explorer

### LINKED ISSUE #264: Explorer does not indicate which file is loaded and can't handle two files with the same label.
# Test notes
* Create an explorer with [safetyData/ADAE.csv](https://github.com/RhoInc/viz-library/blob/master/data/safetyData/ADAE.csv) and [safetyData-queries/ADAE.csv](https://github.com/RhoInc/viz-library/blob/master/data/safetyData-queries/ADAE.csv) and give them the same label.
* Verify that a codebook can be generated for each file.

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #264

### CODE REVIEW

**Comment by samussiah** [comment]
@pburnsdata , there's a lot of crap here from prettier.  The two files that actually changes are src/explorer/makeCodebook.js and src/explorer/fileListing/onDraw.js.  Basically the _selected_ data file function wasn't working and additionally the first file doesn't switch over to the codebook view on click.

File: src/explorer/fileListing/onDraw.js [hunk 1]
```diff
@@ -6,21 +6,19 @@ export function onDraw(explorer) {
...
+      .filter(f => f.path === explorer.current.path)
       .classed('selected', true);
 
     //Linkify the labelColumn
     var labelCells = this.table
-      .selectAll('td')
-      .filter(f => f.col == explorer.config.labelColumn)
-      .classed('link', true)
-      .on('click', function() {
-        var current_text = d3select(this).text();
```
Comment (pburnsdata) [inline src/explorer/fileListing/onDraw.js]: is d3select a function ?

**Comment by danedexF5** [comment]
@jwildfire how do I create an explorer?

**Comment by jwildfire** [comment]
Works as expected. Merging.

**Pull request merged**